### PR TITLE
Add AT to ensure 0-miner Clique/IBFT are valid

### PR DESCRIPTION
**Pull request by rain-on**

### CODE REVIEW

File: acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/clique/CliqueZeroValidators.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+import java.io.IOException;
+
+import org.junit.Test;
+
+public class CliqueZeroValidators extends AcceptanceTestBase {
+
+  @Test
+  public void startingWithZeroValidatorsProducesValidCluster() throws IOException {
+    final String[] validators = {""};
```
Comment (jframe) [inline acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/clique/CliqueZeroValidators.java]: why not just {} as is done for the ibft test?

File: acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/clique/CliqueZeroValidators.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import tech.pegasys.pantheon.tests.acceptance.dsl.node.PantheonNode;
+
+import java.io.IOException;
+
+import org.junit.Test;
+
+public class CliqueZeroValidators extends AcceptanceTestBase {
+
+  @Test
+  public void startingWithZeroValidatorsProducesValidCluster() throws IOException {
```
Comment (jframe) [inline acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/clique/CliqueZeroValidators.java]: perhaps use shorter test name "zeroValidatorsFormValidCluster" that was used in the Clique test

File: acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/clique/CliqueZeroValidators.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    final String[] validators = {""};
+    final PantheonNode node1 = pantheon.createCliqueNodeWithValidators("node1", validators);
+    final PantheonNode node2 = pantheon.createCliqueNodeWithValidators("node2", validators);
+    cluster.start(node1, node2);
+
+    node1.verify(net.awaitPeerCount(1));
```
Comment (jframe) [inline acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/clique/CliqueZeroValidators.java]: should validate both nodes have correct peer count, so cluster.verify(net.awaitPeerCount(1))

File: acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/ibft2/Ibft2ZeroValidators.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+  @Test
+  public void zeroValidatorsFormValidCluster() throws IOException {
+    final String[] validators = {};
+    final PantheonNode node1 = pantheon.createIbft2NodeWithValidators("node1", validators);
+    final PantheonNode node2 = pantheon.createIbft2NodeWithValidators("node2", validators);
+
+    cluster.start(node1, node2);
+
+    node1.verify(net.awaitPeerCount(1));
```
Comment (jframe) [inline acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/ibft2/Ibft2ZeroValidators.java]: should validate both nodes have correct peer count, so cluster.verify(net.awaitPeerCount(1))

File: acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/clique/CliqueZeroValidators.java (hunk 1 above)
Comment (rain-on) [reply acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/clique/CliqueZeroValidators.java]: done.

File: acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/clique/CliqueZeroValidators.java (hunk 2 above)
Comment (rain-on) [reply acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/clique/CliqueZeroValidators.java]: done.

File: acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/clique/CliqueZeroValidators.java (hunk 3 above)
Comment (rain-on) [reply acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/clique/CliqueZeroValidators.java]: done.

File: acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/ibft2/Ibft2ZeroValidators.java (hunk 4 above)
Comment (rain-on) [reply acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/ibft2/Ibft2ZeroValidators.java]: done.

**Pull request merged**